pets-app/src/ThemeContext.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useContext, useState} from 'react';
+
+const ThemeContext = React.createContext(false);
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const ThemeUpdateContext = React.createContext(()=>{});
+
+export function useTheme() {
+  return useContext(ThemeContext);
+}
+
+export function useThemeUpdate () {
+  return useContext(ThemeUpdateContext);
+}
+
+export function ThemeProvider({ children }: any) {
+  const [lightTheme, setDarkTheme] = useState(true);
+
+  function toggleTheme() {
+    setDarkTheme((prevDarkTheme) => !prevDarkTheme);
+  }
+
+  return (
+    <ThemeContext.Provider value={lightTheme}>
```

Comment:
Здесь в value передать два значения
```suggestion
    <ThemeContext.Provider value={{ theme, updateTheme: setTheme }>
```